src/shared/styles/theme.styles.js

```diff
@@ -8,9 +8,11 @@ const theme = {
     grayDark: '#000000',
     grayLight: '#ffffff',
     grayBase: '#93918f',
+    grayPrefix: '#9e9e9e',
```

Comment:
Suggestion: You could start naming the colors `grayMediumLight` or so. This ties it very much to the use-case.

Reply:
Thank you and noted!